pages/account/manage/links.js

```diff
@@ -112,7 +112,7 @@ export default function ManageLinks({ BASE_URL, username, links }) {
           </Button>
 
           {linkList.length !== 0 && !reorder && (
-            <Button onClick={() => setReorder(true)}>
+            <Button onClick={() => setReorder(true)} disable={linkList.length < 2}>
```

Comment:
Maybe you could add `disabled` to the reorder button on the disable condition, so that it's easier to figure out that the reorder button is disabled, along with making the pointer events to be none.

Refer the below syntax for details

```
     <button type="button" className="some tailwind classes" disabled>Button</button>
```

### This is how the disabled button will look after the change (ofcourse, pointer events will still remain none)
![image](https://github.com/EddieHubCommunity/BioDrop/assets/111000515/893ce2e6-68a7-4373-b64c-5cfcb0c8c248)

Reply:
Button is a custom component defined in `components/Button.js` which is imported in `manage/links.js` and `disable` is a prop not html attribute.
And using `disabled` directly on `<button> in components/Button.js` will not check condition.